components/carousel/carousel.styl

```diff
@@ -0,0 +1,34 @@
+.carousel-track__image
+  width 100%
+  margin-bottom 40px
+
+.carousel-figure-img
+  width 100%
+  height 54vw
+  background-size cover
+  background-position center center
+
+figure.slide-images
+  display none
+  width 100%
+  margin 0px
+
+figure.slide-images div
```

Comment:
I think you have the specificity reversed here :) I would do something like:
`.slide-images .caption` (if its a caption)
Easier to read and understand quickly.